CONTRIBUTING.md

```diff
@@ -1,218 +1,88 @@
-at [CHAOSS Slack](https://join.slack.com/t/chaoss-workspace/shared_invite/zt-r65szij9-QajX59hkZUct82b0uACA6g)
-
-We use the channel for pinging people, sharing updates, quick and informal
-discussions, questions and answers, etc. Please, don't consider that
-developers in the channel will be always available, or always willing
-to answer questions and comments. However, anyone interested in the
-project is welcome to join the channel and hang around.
-
-### Pull request / issues
-
-Repositories: [GrimoireLab repositories in GitHub](http://github.com/chaoss/grimoirelab), which are all in the [CHAOSS organization](http://github.com/chaoss).
-
-Most of the work is discussed here, including upgrades and
-proposals for upgrades, bug fixing and feature requests. For any of
-these, open an issue in the corresponding repository. If in doubt, ask in
-the IRC channel, or try in any one: if needed you will be redirected to
-the right repository. If you are proposing code change, open a pull request.
-
-## DCO and Sign-Off for contributions
-
-The [CHAOSS Charter](https://github.com/chaoss/governance/blob/master/project-charter.md) requires that contributions
-are accompanied by a [Developer Certificate of Origin](http://developercertificate.org) sign-off.
-For ensuring it, a bot checks all incoming commits.
-
-For users of the git command line interface, a sign-off is accomplished with the `-s` as part of the commit command: 
-
-```
-git commit -s -m 'This is a commit message'
-```
-
-For users of the GitHub interface (using the "edit" button on any file, and producing a commit from it),
-a sign-off is accomplished by writing
-
-```
-Signed-off-by: Your Name <YourName@example.org>
-```
-
-in a single line, into the commit comment field. This can be automated by using a browser plugin like
-[DCO GitHub UI](https://github.com/scottrigby/dco-gh-ui).
-
-## Committers
-
-Committers have the permision to merge code in GrimoireLab. Committers
-can be for all GrimoireLab, or for specific repositories.
-
-New committers will be proposed by committers, and accepted (granted
-committer rights) by an action vote among committers (see below).
-Developers can apporach committers to ask to be proposed as comitter.
-Main rule for acceptance of new committers will be their merits on the
-project, including past contributions and expertise.
-
-Committers may resign if they are no longer involved enough in the
-project. Committers can also propose the removal of other committer
-rights, in case they are no longer involved in the project. For being
-removed, after the proposal there will be an action voting among
-committers, except for the one being proposed to be removed.
-
-The current list of committers is (GitHub handles):
+If you have ideas for improvements or new features, feel free to open a new
+discussion or issue to share them. We appreciate well-documented
+enhancement suggestions. You can also have at look to our [ROADMAP](./ROADMAP.md)
```

Comment:
The ROADMAP.md file does not exist. Is it something to add in the future?

Reply:
Yes, I have it ready and I will upload it later today after this PR is merged.